docs/src/app/components/pages/components/buttons.jsx

```diff
@@ -58,9 +58,9 @@ export default class ButtonPage extends React.Component {
             desc: 'Override the inline hover color of the button\'s root element.',
           },
           {
-            name: 'label or children',
-            type: 'string (label) or HTML/React elements (children)',
-            header: 'required',
+            name: 'label',
+            type: 'string',
+            header: 'optional',
```

Comment:
I took some time to read the source code. I would not change it.
Seems like we have two options here to do the same thing. IMO it's better to only have one option.
May worse to kill label in the future.